src/ansys/mapdl/core/misc.py

```diff
@@ -544,13 +544,9 @@ def get_name():
     # create dir:
     path = os.path.join(tmpdir, name)
 
-    try:
-        os.mkdir(path)
-    except:  # pragma: no cover
-        raise MapdlRuntimeError(
-            "Unable to create temporary working "
-            f"directory {path}\nPlease specify 'run_location' argument"
-        )
+    if not os.path.isdir(path):
+        os.makedirs(path, exist_ok=True)
```

Comment:
why `exist_ok=True` if this code only runs if `os.path.isdir` returns `False`?

Reply:
You are right. It is not needed.